stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2883,6 +2945,7 @@ internal struct _Native${Self}Storage<${TypeParameters}> {
     return _buffer._initializedHashtableEntriesBitMapStorage
   }
 
+  // just a convenience for _keys and _values
```

Comment:
This comment doesn't tell me what the thing *is*, and its implementation is sufficiently scary that I want to know.